core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -53,7 +59,7 @@ trait PartitionStateStore {
 
 class ZkPartitionStateStore(topicPartition: TopicPartition,
                             zkClient: KafkaZkClient,
-                            replicaManager: ReplicaManager) extends PartitionStateStore {
+                            isrChangeListener: IsrChangeListener) extends PartitionStateStore {
```

Comment:
Maybe we can move this out of `ZkPartitionStateStore` since we need the dependence on `IsrChangeListener` in `Partition` anyway.